package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless",
-  "version": "1.50.0",
```

Comment:
We could keep the old version here to avoid cherry-picking as @medikoo mentioned in his PR review.